redash/models/__init__.py

```diff
@@ -362,9 +362,21 @@ def get_latest(cls, data_source, query, max_age=0):
         query_hash = gen_query_hash(query)
 
         if max_age == -1:
```

Comment:
It seems that the front end only triggers a query with max_age = -1 when there are no search results. In this case, if there are already search results, do we still need to manually refresh?

Reply:
> It seems that the front end only triggers a query with max_age = -1 when there are no search results. In this case, if there are already search results, do we still need to manually refresh?

When there is a result, max_age=-1 will also be triggered, but for example, if I want to see tomorrow's data tomorrow instead of the data queried yesterday, what should I do? At present, it will also use max_age to find old query results

Reply:
> @junnplus 我尝试用中文和你交流一下吧！现在 redash 如果某个查询已经有了查询结果了，那么它以后都会一直获取到这个查询结果，如果 sql 不改的话，现在遇到了问题，比如我的数据库的数据每天都会有新的数据进来，那么我过了一天想获取到新的查询结果，而不是之前旧的数据，现在的 redash 一直会 get 到旧的数据，所以想加一个自己配置一个结果的失效时间，那么我下次打开这个查询的话如果结果超过了我的配置 ttl 时间，那么它就会去主动去查询新的结果集。感谢你的时间和回复。我不知道表达清楚没有我的解释。

每次打开有结果的查询它都会用 `max_age == -1` 这个去获取结果, 当条件为 `max_age != -1` 的话就代表这个查询没有查询过结果或者是刷新操作。

Reply:
Thank you for your detailed explanation. I was wondering if we could achieve this by adding this configuration to the front-end instead of passing max_age = -1 every time.

Reply:
> Thank you for your detailed explanation. I was wondering if we could achieve this by adding this configuration to the front-end instead of passing max_age = -1 every time.


Not thought of a better solution yet.